image-builder/builder-services/src/main/java/io/hyscale/builder/services/impl/LocalImagePushServiceImpl.java

```diff
@@ -92,11 +95,6 @@ public void pushImage(ServiceSpec serviceSpec, BuildContext buildContext) throws
 
         tagImage(sourceImage, imageFullPath);
 
-        if (buildContext.getImageRegistry() == null) {
```

Comment:
How are we printing the image push skipped message where we don't need to push the image?